### ISSUE TITLE
GB Donation panel doesn't adjust Support ratio in 194% 195% and + threads

### ISSUE THREAD

**Issue by kronikpillow** [FoE-Info/FoE-Info-Extension#35]
GB Donation panel doesn't adjust Support ratio in 194% 195% and + threads, while it should ..
![194% not adjusted](https://i.imgur.com/8XlZ3HW.png)

althou it does adjust Correctly in 190 191 192% support threads ...

desired behavior
if the thread name contains 190%-200% anywhere in its name, automatically adjust highlighted calculation to that ratio
also include 1.90 1.91 1.92-2.0 variants as well so that it would recognize such threads as well ...

didn't find any errors in debug log

**Comment by kronikpillow** [comment]
further investigating on the issue ... following conclusion is reached ...

Guild Tab:
if title contains 1.97 ratio doesn't get adjusted
if title contains 1.99 ratio doesn't get adjusted
if title contains 2.0  ratio doesn't get adjusted

Social Tab:
if title contains 1.97 ratio doesn't get adjusted
if title contains 1.99 ratio doesn't get adjusted
if title contains 2.0  ratio doesn't get adjusted

adding any pre-text to Title breaks auto-adjustment
adding pre-text to ratio like:
[LoW]🔓 194% Arcs only🔓
[LoW]🔓 195% Arcs only🔓
doesn't adjust the ratio in the panel

while if 19+% is the first word
🔓 194% Arcs only🔓 [LoW]
🔓 195% Arcs only🔓 [LoW]
adjusts ratio

desired behavior:
adjust ratio regardless of pre-text or post text, just search for 1.90+ or 19+% word

**Comment by kronikpillow** [comment]
while adding any kind of pre-text except emoji to the Ratio in Title breaks auto-adjustment

adding pre-text to ratio like:
```
[LoW]🔓 194% Arcs only🔓
[LoW]🔓 195% Arcs only🔓
```
doesn't adjust the ratio in the panel

while if 19+% is the first word
```
🔓 194% Arcs only🔓 [LoW]
🔓 195% Arcs only🔓 [LoW]
```
adjusts ratio

desired behavior:
adjust ratio regardless of pre-text or post text, just search for 1.90+ or 19+% word

**Comment by kronikpillow** [comment]
fixed in merge request #40

**Comment by kronikpillow** [comment]
![not supported](https://i.imgur.com/6L4PNFm.png)

I found that the following format was not supported

**Comment by kronikpillow** [comment]
solved with #41

**Comment by kronikpillow** [comment]
actually still not solved

the following formats are for some reason using 1.9 ratio ...

![image1](https://cdn.discordapp.com/attachments/641226916661166080/1054374197704544267/image.png)
![image2](https://cdn.discordapp.com/attachments/641226916661166080/1054374634419666994/image.png)
